electron/app/locales/en/webui.json

```diff
@@ -1186,7 +1186,7 @@
   "k8s-domain-status-checker-domain-status-unknown": "Unknown - Cannot determine the domain status now - check back later",
   "k8s-domain-status-checker-domain-status-complete": "Complete - Domain is deployed and all servers are up and running",
   "k8s-domain-status-checker-domain-status-failed": "Failed - Domain failed to deploy. Reason: {{reason}}",
-  "k8s-domain-status-checker-domain-status-available": "Available - Domain is deployed and some servers are up and running but the domain has not reaching the replicas count goal yet. Reason: {{reason}}",
+  "k8s-domain-status-checker-domain-status-available": "Available - Domain is deployed and some servers are up and running but the domain has not reached the replicas count goal yet, check again later.",
```

Comment:
Hmm...  In most cases, the reason is available and presumably helps the user quickly and easily understand the state.  I don't feel like the solution to handling the transient situations where the reason is empty is to completely eliminate showing the reasons in all situations.

Reply:
changed the code to put out condition reasons